Solved issue #522 (#623)
python/devguide: documenting.rst

I have a translation, but not on git, what should I do?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> Ask for help on the doc-sig mailing list and our python console, git

Reviewer: ```suggestion
Ask for help on the doc-sig mailing list
```

> and bash command line interface combined will help you create an appropriate repository. 

Reviewer: ```suggestion
about creating an appropriate repository. 
```

> Using a tool like transifex can help keeping them in sync.

Reviewer: ```suggestion
Using a tool like transifex can help keep translations in sync.
```
